staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -750,14 +748,6 @@ func TestPriorityAndFairnessWithPanicRecoveryAndTimeoutFilter(t *testing.T) {
 		case <-time.After(wait.ForeverTestTimeout):
 			t.Errorf("Expected the server handler to have completed: %q", secondRequestPathShouldWork)
 		}
-
-		close(stopCh)
-		t.Log("Waiting for the controller to shutdown")
-
-		controllerErr := <-controllerCompletedCh
-		if controllerErr != nil {
-			t.Errorf("Expected no error from the controller, but got: %#v", controllerErr)
```

Comment:
should we remove this assertion? this assertion verifies that the controller did not yield any error. The test runs the controller and tracks the error it returns, unlike the post start hook.  I should note that this PR fixes the post start hook issue.

Reply:
Yeah, the only place this could be triggered was when waiting for the informer. All this just started with the linter complaining about not handling the error, sending me into this rabbit hole 🐰:D